meson.build

```diff
@@ -210,7 +210,8 @@ if get_option('build_backends')
 			    '@BASENAME@_avx2.obj' ]
 	endif
       else
-        outputnames = [ '@BASENAME@.o']
+	ispc_extra_args += ['--pic']
```

Comment:
It seems that this part of the meson build file has weird indentation. Would it be possible for you to fix it so that it's clear what matches what?

Reply:
There were tabs and spaces in mix for ispc part of meson code. Last patch for this PR a443c44 should fix this.